stdlib/builtins.pyi

```diff
@@ -1952,7 +1952,7 @@ if sys.version_info >= (3, 11):
         @overload
         def subgroup(self: Self, __condition: Callable[[_BaseExceptionT_co], bool]) -> Self | None: ...
         @overload
-        def split(
+        def split(  # type: ignore[misc]  # complaints about overlapping overloads
```

Comment:
I believe these new mypy errors are already fixed on mypy master, by https://github.com/python/mypy/pull/13461. Unfortunately, it doesn't look like that PR made it into 0.981 (cc. @hauntsaninja, @ilevkivskyi)

Reply:
cc @jhance if people will complain about this, we will need to CP it and release 0.982.

Reply:
@Molkree thanks!

Reply:
Mypy was successfully bumped to 0.982 in #8831; the `type ignore`s are gone again 🎉